includes/import/class-wc-product-csv-importer.php

```diff
@@ -24,6 +24,13 @@
 class WC_Product_CSV_Importer extends WC_Product_Importer {
 
 	/**
+	 * The file position after the last read.
+	 *
+	 * @var int
+	 */
+	protected $file_position = 0;
```

Comment:
Better pass this and the other changes (excluding the change from `read_file()`, of course) the `WC_Product_Importer` abstract class to make easy to use the same code for other importers.